templates/repo.tmpl

```diff
@@ -3,6 +3,7 @@ name = {{ name }}
 enabled={% if enabled %}1{%else%}0{%endif%}
 gpgcheck={% if gpgcheck %}1{%else%}0{%endif%}
 baseurl = {{ baseurl }}
+mirrorlist = {{ mirrorlist }}
```

Comment:
Does this work as expected if `mirrorlist == null`? Same for `baseurl`.

Reply:
I think so, but it might be better to leave it out of the config file in that case.

Reply:
I just tried it on a dummy module and it seems to result in the literal string "None". So this would  become "mirrorlist = None", which isn't valid, is it?